Update the RPM URI to refer to GitHub, not RedHat.

RokeJulianLockhart opened the pull request:
Updates the RPM specification URI to refer to this GitHub repository, rather than RedHat. I don't know whether a “Bug URL” key would also be useful.

RokeJulianLockhart committed:
Update RPM URI to refer to GH, not RH.

Updates the RPM specification URI to refer to this GitHub repository, rather than RedHat.

Signed-off-by: Mr. Beedell, Roke Julian Lockhart (RJLB) <wy46xghh@rokejulianlockhart.addy.io>

beedell-roke commented on virtio-win.spec:
```diff
@@ -47,7 +47,7 @@ Name: virtio-win
 Release: 1
 Group: Applications/System
-URL: http://www.redhat.com/
+URL: https://github.com/virtio-win/virtio-win-pkg-scripts
```
It's more useful when the domain of each URI differs. I don't see much value in having both when `/issues` is so trivial to locate.

kostyanf14 replied:
I agree to ignore this comment.

YanVugenfirer replied:
https://github.com/virtio-win/virtio-win-pkg-scripts is not really project URL. This is packaging scripts project URL, not the drivers project URL.


https://github.com/virtio-win/ better reflects project's location.

beedell-roke replied:
@YanVugenfirer, `virtio-win` contains `virtio-win.github.io`, etcetera. If a user is using the RPM, they generally want to report bugs to `virtio-win-pkg-scripts`, and even if they don't, navigating from `virtio-win-pkg-scripts` to `virtio-win` is trivial.

However, if I linked to `virtio-win`, discovering where to report issues — `virtio-win-pkg-scripts` — would be more difficult. Consequently, if I were to incorporate your suggestion, I would advise that we add the `/issues` recommendation as a separate `BugURL` line. However, I don't see why we would do this.

---

For subsequent responses, observe [`#issuecomment-3884469436`](https://github.com/virtio-win/virtio-win-pkg-scripts/pull/114#issuecomment-3884469436).

YanVugenfirer commented:
Reporting bugs to virtio-win-pkg-scripts is only for the installer related issues (also not recommended).

Driver bugs and actually any issues in general should be reported to https://github.com/virtio-win/kvm-guest-drivers-windows/issues .

RokeJulianLockhart commented:
@YanVugenfirer, in which case, should we replace the URI with that? I'll gladly do that.

YanVugenfirer commented:
@RokeJulianLockhart Sure

RokeJulianLockhart committed:
Replace the URI with KVM Guest Drivers issues'

Updates the URI, in the `virtio-win.spec` file, to direct to `kvm-guest-drivers-windows/issues`, as https://github.com/virtio-win/virtio-win-pkg-scripts/pull/114#issuecomment-3898092115 advises.

Signed-off-by: Mr. Beedell, Roke Julian Lockhart (RJLB) <wy46xghh@rokejulianlockhart.addy.io>
